t/lib/OpenQA/Test/Utils.pm

```diff
@@ -174,12 +174,18 @@ sub wait_for_worker {
 }
 
 sub create_webapi {
-    my $mojoport = shift;
+    my $mojoport    = shift // Mojo::IOLoop::Server->generate_port;
+    my $schema_hook = shift // undef;
     note("Starting WebUI service. Port: $mojoport");
 
     my $startingpid = $$;
     my $mojopid     = fork();
     if ($mojopid == 0) {
+        if ($schema_hook) {
```

Comment:
I believe we must create default schema object if $schema_hook is null: otherwise there is no way to create schema in the fork?

Reply:
Jein. We have tests that use no schema, and others which do. But you're raising a very good point here, in that it's not very obvious. I'll make this explicit in all cases so the test code is more discoverable in all cases.